website/pages/en/index.js

```diff
@@ -74,17 +73,11 @@ class HomeSplash extends React.Component {
 class Index extends React.Component {
   render() {
     let language = this.props.language || 'en';
-    const showcase = siteConfig.users
-      .filter(user => {
-        return user.pinned;
-      })
-      .map((user, i) => {
-        return (
-          <a href={user.infoLink} key={i}>
-            <img src={user.image} alt={user.caption} title={user.caption} />
-          </a>
-        );
-      });
+    const showcase = siteConfig.users.filter(user => user.pinned).map(user => (
```

Comment:
Instead of filter and then map, I think using `reduce` can be a good idea

Reply:
I'd have to use an `if` inside `reduce`, which makes the code less declarative, so I don't think it's a good idea.